astropy/coordinates/tests/test_arrays.py

```diff
@@ -100,29 +100,28 @@ def test_array_coordinates_creation():
     """
     Test creating coordinates from arrays.
     """
-    from .. import Angle
-    from .. import ICRS, CartesianRepresentation
+    from .. import Angle, SkyCoord, CartesianRepresentation
 
-    c = ICRS(np.array([1, 2])*u.deg, np.array([3, 4])*u.deg)
+    c = SkyCoord(np.array([1, 2])*u.deg, np.array([3, 4])*u.deg)
     assert not c.ra.isscalar
```

Comment:
I would think about specific frame tests going into `test_frame.py`.  This already has a test like the first of these three (https://github.com/astropy/astropy/blob/master/astropy/coordinates/tests/test_frames.py#L173), so one only needs to add the second and third.  And actually just looking now there are certainly tests in `test_frame.py` of initializing from a representation, just not (obviously) initializing from a cartesian one.